README.md

```diff
@@ -157,7 +157,9 @@ Event          | Description
 `markerEndDragging` | Fires when a marker ends dragging
 `markerInfoWindowTapped` | Fired on tapping Marker Info Window
 `cameraChanged` | Fired after the camera has changed
+`cameraMoveStarted` | Fired while the camera start move
 `cameraMove` | Fired while the camera is moving
+`cameraMoveStoped` | Fired while the camera stop move
```

Comment:
`cameraMoveStopped` -> Little typo

Also, in your code it's named `cameraMoveCanceled` and not `stopped`

Reply:
Hi @RignonNoel I made the corrections, but kept the term "canceled" due to its use in the android api.